### DISCUSSION TITLE
Symbol mapping?

### DISCUSSION THREAD

**Discussion by lsalgueiro** [ghostfolio/ghostfolio#2452] [Q&A]
I'm a European trader, and I'm trying to import my portfolio from IBKR into Ghostfolio.

IBKR's Flex report does not output symbols that are immediately parsable by Yahoo — namely, European ones, which need a suffix for their exchange to be recognized.

This in itself would be a feature request, but I've noticed a "Symbol Mapping" field in Admin/Market Data. There is no documentation to go on, but I assumed this would allow me to alias IBKR's output to an existing entry, allowing it to be correctly parsed. However, entering a value (I've tried `{"XZY"}`) and saving does not save the input.

Is this a planned feature? Am I misunderstanding something?

**Comment by dtslvr** [comment]
Hi @lsalgueiro

The symbol mapping concept is currently only supported for data enhancers. You need to transform your symbols (tickers) manually before the import.
